docs/experimental-organization.md

```diff
@@ -271,17 +280,144 @@ Plots: geometrical objects such as polygons, provided with Well-known text stand
+This can be achieved by clicking on the <span class="btn btn-success">**+**</span> green icon on the right-hand side of the **Unit label** field.
+In the case of a new unit, do not fill the `Unit label` field but the `Internal label` one, below the <span class="btn btn-danger">**-**</span> red icon that replaced the <span class="btn btn-success">**+**</span> green icon.
+This new unit label should be if possible meaningful, distinct from other unit labels, and underscores "\_" should be avoided.
+
+Otherwise, if the appropriate unit is already register in Phis, then one only has to select it from the `Unit label` field, without clicking on the <span class="btn btn-success">**+**</span> green icon (or clicking on the <span class="btn btn-danger">**-**</span> red icon if the <span class="btn btn-success">**+**</span> green icon had previously been clicked on).
+
+![existing-trait](img/create-variable_existing-unit.png)
+
+`Related References`.
+In order to unambiguously define the new variable, semantic relations are established by the user between the new variable and concepts already defined in reliable ontologies existing on the world wide web.
+These relations to external ontologies are established using **SKOS** (Simple Knowledge Organization System).
+`Entity` refers to the Phis entity that will be associated to a concept found in an ontology.
+This **Entity** be either the new variable, trait, method or unit.
+It is not required to specify related concepts for traits, methods and units, however it is recommanded.
+`Relation` refers to the nature of the relation between the entity and the concept defined in an ontology.
+This semantic relation is provided using SKOS.
+The entity can either be, compared to an ontology concept, an **exact match**, a **close match**, **narrower** or **broader**.
+These SKOS semantic relations are explicited at this [w3 web page](https://www.w3.org/TR/skos-reference/#semantic-relations).
+`Reference URI` refers to the URI of the concept found on ontologies such as the ones suggested in the short list above the **Related References** field.
+Do not erroneously provide in this field the URL of the web page of the ontology on which the targeted concept is defined.
+Indeed, the URI of a given concept does not necessarily match with the URL of the web page where this concept has been found.
+`Hyperlink` (optionnal) refers to the URL where are located the related concept whose URI has been provided in the previous field.
+
+When a variable is created, multiple references using SKOS can be stated, using the <span class="btn btn-default">**+**</span> white icon.
+
+![add-relation-to-a-reference](img/create-variable_add-relation.png)
+
+The main ontologies differ on the following features:
+
+The **AgroPortal** project aims to offer a reference ontology repository for agronomy, reusing the NCBO BioPortal technology.
```

Comment:
add links to the web pages : 
- agroportal
- agrovoc
- plant ontology
- planteome
- crop ontology
- unit ontology